tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Solution/ListSolutionsSteps.cs

```diff
@@ -87,8 +87,8 @@ public async Task ThenTheSolutionsAreNotFoundInTheResponse(List<string> solution
         [Then(@"the details of the solutions returned are as follows")]
         public async Task ThenTheDetailsOfTheSolutionsReturnedAreAsFollows(Table table)
         {
-            var expectedSolutions = table.CreateSet<SolutionDetailsTable>();
-            var solutions = (await _response.ReadBody().ConfigureAwait(false)).SelectToken("solutions");
+            var expectedSolutions = table.CreateSet<SolutionDetailsTable>().ToList();
+            var solutions = (await _response.ReadBody()).SelectToken("solutions");
 
             solutions.Count().Should().Be(expectedSolutions.Count());
```

Comment:
We can use `.Count` rather than `.Count()` now that `expectedSolutions` is a list.

Reply:
👍